app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] Good job adding `update_comments_counter` method, however, it could be better to call the `#update_comments_counter` method every time a new post has been saved to database, then make the `update_comments_counter` method private. it will be something like this:

````ruby
class Comment < ApplicationRecord
  belongs_to :post
  belongs_to :user

  after_save :update_comments_counter

  private

  def update_comments_counter
    post.update(comments_counter: post.comments.count)
  end
end
````